plugin/Controller/AdminViewController.php

```diff
@@ -417,4 +422,27 @@ public function getField()
 	{
 		return new Fieldnames(new FieldsCollection());
 	}
+
+	public function generalAdminNoticeSEO() {
+		$urlOnofficeSetting = admin_url().'admin.php?page=onoffice-settings';
+		$nameOnofficeSetting = esc_html__('onOffice plugin settings','onoffice-for-wp-websites');
+		$pluginOnofficeSetting = sprintf("<a href='%s' target='_blank' rel='noopener'>%s</a>", $urlOnofficeSetting,$nameOnofficeSetting);
+		if (count(array_intersect(["wordpress-seo/wp-seo.php", "seo-by-rank-math/rank-math.php", "wpseo/wpseo.php"], get_option("active_plugins"))) > 0) {
+			if (get_option('onoffice-click-button-close-action') == 0
+				&& get_current_screen()->id !== 'onoffice_page_onoffice-settings'
+				&& get_option('onoffice-settings-title-and-description') == 0) {
+				echo "<div class='notice notice-warning active-plugin-seo is-dismissible'>
+						<p> " . esc_html__('The onOffice plugin has detected an active SEO plugin: Yoast SEO.', 'onoffice-for-wp-websites') . "
+							" . esc_html__('You currently have configured the onOffice plugin to fill out the title and description of the detail page, which can lead to conflicts with the SEO plugin.', 'onoffice-for-wp-websites') . "<br>
+							" . esc_html__('We recommend that you go to the', 'onoffice-for-wp-websites') . "
```

Comment:
There are a few improvements I would like to make.

1. It is difficult to translate each sentence separately. Additionally, it is hard to control the placement of the `$pluginOnofficeSetting`. It would be better to put it all in one `esc_html__` and to use `sprintf` to place the `$pluginOnofficeSetting`. That way a translator can place it where they need to.

    Example:
    ```
    sprintf(esc_html__('We recommend that you go to the %s and configure the onOffice plugin to not modify the title and description. This allows you to manage the title and description with your active SEO plugin.', 'onoffice-for-wp-websites'), $pluginOnofficeSetting);
    ```

    Since we need to handle line breaks (and in the other comment you can see that it would be nice to handle lists as well), consider using a Markdown parser like https://github.com/erusev/parsedown. First, translate the text, then put it through the parser. For example:

    ```
    echo Parsedown::instance()->text(
      sprintf(
        esc_html__('We recommend that you go to the %s and configure the onOffice plugin to not modify the title and description. This allows you to manage the title and description with your active SEO plugin.', 'onoffice-for-wp-websites'),
        $pluginOnofficeSetting
      )
    );
    ```

2. The warning should identify the conflicting plugin. Currently it always says "Yoast SEO".

    The desired display names are:
    - `wordpress-seo/wp-seo.php` -> "Yoast SEO"
    - `seo-by-rank-math/rank-math.php` -> "Rank Math SEO"
    - `wpseo/wpseo.php` -> "wpSEO"